src/main/java/com/javadiscord/javabot/service/help/HelpChannelManager.java

```diff
@@ -245,26 +269,24 @@ private void unreserveChannelByOwner(TextChannel channel, User owner, Interactio
 	private void sendThanksButtonsMessage(List<Member> potentialHelpers, ChannelReservation reservation, Interaction interaction, TextChannel channel) {
 		List<Component> thanksButtons = new ArrayList<>(25);
 		for (var helper : potentialHelpers.subList(0, Math.min(potentialHelpers.size(), 20))) {
-			thanksButtons.add(new ButtonImpl("help-thank:" + reservation.getId() + ":" + helper.getId(), helper.getEffectiveName(), ButtonStyle.SUCCESS, false, Emoji.fromUnicode("❤")));
+			thanksButtons.add(new ButtonImpl("help-thank:" + reservation.getId() + ":" + helper.getId(), helper.getEffectiveName(), ButtonStyle.SECONDARY, false, Emoji.fromUnicode("❤")));
 		}
 		ActionRow controlsRow = ActionRow.of(
-				new ButtonImpl("help-thank:" + reservation.getId() + ":done", "Unreserve", ButtonStyle.PRIMARY, false, Emoji.fromUnicode("✅")),
-				new ButtonImpl("help-thank:" + reservation.getId() + ":cancel", "Cancel", ButtonStyle.SECONDARY, false, Emoji.fromUnicode("❌"))
+				Button.success("help-thank:" + reservation.getId() + ":done", "Unreserve"),
+				Button.danger("help-thank:" + reservation.getId() + ":cancel", "Cancel")
 		);
-		interaction.getHook().sendMessage("Before your channel is unreserved, we would appreciate if you could take a moment to acknowledge those who helped you. This helps us to reward users who contribute to helping others, and gives us better insight into how to help users more effectively. Otherwise, click the **Unreserve** button simply unreserve your channel.")
-				.setEphemeral(true).queue();
 		List<ActionRow> rows = new ArrayList<>(5);
-		rows.addAll(MessageActionUtils.toActionRows(thanksButtons));
 		rows.add(controlsRow);
-		channel.sendMessage(THANK_MESSAGE_TEXT).setActionRows(rows).queue();
+		rows.addAll(MessageActionUtils.toActionRows(thanksButtons));
+		interaction.getHook().sendMessageEmbeds(getHelpChannelEmbed(config.getThankMessage(), null)).addActionRows(rows).queue();
 	}
 
 	private void unreserveChannelByOtherUser(TextChannel channel, User owner, @Nullable String reason, Interaction interaction) {
 		if (reason != null) {// The user provided a reason, so check that it's legit, then send a DM to the owner.
 			if (reason.isBlank() || reason.length() < 5) {
 				Responses.warning(interaction.getHook(), "The reason you provided is not descriptive enough.").queue();
 			} else {
-				Responses.info(interaction.getHook(), "Channel Unreserved", "The channel has been unreserved.").queue();
+				Responses.info(interaction.getHook(), null, "The channel has been unreserved.").queue();
```

Comment:
Why remove the title? (And comment from above applies regarding nullability of parameters in Responses methods)